ext/console/02_console.js

```diff
@@ -624,10 +641,9 @@
         }
 
         if (ctxHas(value)) {
-          // Circular string is cyan
-          return cyan("[Circular]");
+          return handleCircular(value);
         }
-        return inspectObject(value, level, inspectOptions);
+        return inspectObject(value, level, inspectOptions, circular);
```

Comment:
The 4th arg doesn't seem being used?

Reply:
Whoops, that was remainder from some testing